packages/app/src/systems/Account/machines/accountMachine.tsx

```diff
@@ -141,7 +141,7 @@ export const accountMachine = createMachine(
           const accounts = await AccountService.getAccounts();
           const account = accounts[0];
           if (!account) {
-            throw new Error('Account not found');
+            return undefined;
```

Comment:
The state of not having an account should be handled by the cond `hasAccount`, but if it throws a Error here, it will never go to the next steps of the machine. Throwing an error when you start the machine but don't have an account already set up.

Reply:
I prefer throwing the error returning undefined is never a great thing